pyracf/common/security_admin.py

```diff
@@ -46,6 +46,12 @@ class SecurityAdmin:
         "base:gaudupdt": "racf:gaudupdt",
         "base:generic": "racf:generic",
     }
+
+    _secret_traits = {
```

Comment:
I don't think that `_secret_traits` gets used anywhere else besides `security_admin.py`. In that case, it should start with `__` so that it can't be used anywhere outside `security_admin.py` including child classes. So, this field should be changed to `__secret_traits`.

Reply:
Confirmed that `_secret_traits` is only used in `security_admin.py`. Change made.